Meta/import-wpt-test.py

```diff
@@ -81,23 +82,26 @@ def download_files(filepaths):
     downloaded_files = []
 
     for file in filepaths:
-        if (file.destination.exists()):
-            print(f"Skipping {file.destination} as it already exists")
+        source = urljoin(file.source, "/".join(file.source.split('/')[3:]))
+        destination = Path(file.destination).resolve()
```

Comment:
I think using `absolute()` would be slightly better here, since  it won't attempt to do any normalization or resolution of symlinks.

Reply:
> I think using `absolute()` would be slightly better here, since it won't attempt to do any normalization or resolution of symlinks.

✅  Made it so